tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -53,6 +53,13 @@ EnclaveData::EnclaveData(void) {
     SerializePublicData();
 }
 
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+EnclaveData::~EnclaveData(void) {
+    serialized_private_data_.clear();
+    serialized_public_data_.clear();
```

Comment:
Why only these variable, class has other instance variables are also.

Reply:
I tried calling destructors of other class members which are inturn objects of different class (related to crypto), but enclave seems to be crashing. Hence i tried not calling destructor explicitly.

Reply:
Please add a comment about that in the code.

Reply:
Ok